src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -2251,7 +2256,9 @@ ucs_status_t uct_ib_mlx5_devx_mem_attach(uct_md_h uct_md,
         goto err;
     }
 
-    memh      = ucs_derived_of(ib_memh, uct_ib_mlx5_devx_mem_t);
+    memh = ucs_derived_of(ib_memh, uct_ib_mlx5_devx_mem_t);
+    uct_ib_mlx5_devx_mem_reset_keys(memh);
```

Comment:
What does the issue look like to the user?
Is it possible to add a test to cover the issue?

Reply:
> What does the issue look like to the user?

Wrong rkey -> remote access error

> Is it possible to add a test to cover the issue?

Will be covered with GGA transport which uses attached memh for RMA

Reply:
Maybe add common function uct_ib_mlx5_devx_memh_alloc() tat will unite the common code in lines 2254-2260 and 760-767?